pyaedt/hfss.py

```diff
@@ -2478,13 +2480,35 @@ def assign_primary(self, face, u_start, u_end, reverse_v=False, coord_name="Glob
         return self._create_boundary(primary_name, props, "Primary")
 
     def _create_pec_cap(self, sheet_name, obj_name, pecthick):
-        # TODO check method
-        obj = self.modeler[sheet_name].clone()
+        """Create a PEC object to back a wave port.
+
+
+        Parameters
+        ----------
+        sheet_name : str
+            Name of the 3d object touching the port surface.
+        obj_name : ???
+        pecthick : float
+            Thickness of the PEC cap
```

Comment:
```suggestion
            Thickness of the PEC cap.
```
I know you'll add a data object link for the previous parameter.